### ISSUE TITLE
Streaming Service API Endpoint is incorrect in new region

### ISSUE THREAD

**Issue by shukiku** [oracle/oci-go-sdk#172]
I faced the error below when I used terraform oci provider resource "oci_streaming_stream" in ap-tokyo-1.

```
Error: Post https://streams.ap-tokyo-1.streaming.oci.oraclecloud.com/20180418/streams: dial tcp: lookup streams.ap-tokyo-1.streaming.oci.oraclecloud.com: no such host
```

Looking at the doc, I see the Streaming Service API Endpoint is `streaming.<region>.oci.oraclecloud.com`.
https://docs.cloud.oracle.com/iaas/api/#/en/streaming/20180418/

So, I guess that the endpoint string at streaming/streaming_streamadmin_client.go should be changed.
https://github.com/oracle/oci-go-sdk/blob/dc0015c7e80a48aed9ad8805b4d8451ca60f72aa/streaming/streaming_streamadmin_client.go#L38-L41

Thanks,

**Comment by eginez** [comment]
@shukiku Thanks for noticing this, we are working on fixing this issue

**Comment by JulienDollon** [comment]
Hello @shukiku 
Thank you for bringing this to our attention. We recently fixed an issue that were generating malformed endpoint for specific regions like Tokyo.
The fix is supposed to be roll out soon (most likely tomorrow).
I will close this bug when it's confirmed.

**Comment by shukiku** [comment]
Hi, @eginez @JulienDollon

Thank you for your response!
I confirmed this issue was fixed in terraform oci provider v3.32.0 (including this fix).

I can manage the "oci_streaming_stream" resouces in ap-tokyo-1.
Thanks again!

**Comment by eginez** [comment]
Indeed this was fixed in #176
